docker/Dockerfile.linux.arm64

```diff
@@ -1,7 +1,7 @@
 FROM arm64v8/alpine:3.15 as alpine
 RUN apk add -U --no-cache ca-certificates
 
-FROM arm64v8/alpine:3.15
+FROM scratch
```

Comment:
Have some common comments between the s3 PR and this one - please refer to that PR once

Reply:
Resolved. Will make this change in a different PR.